great_expectations/data_context/util.py

```diff
@@ -25,13 +26,18 @@
 
 # TODO: Rename config to constructor_kwargs and config_defaults -> constructor_kwarg_default
 # TODO: Improve error messages in this method. Since so much of our workflow is config-driven, this will be a *super* important part of DX.
-def instantiate_class_from_config(config, runtime_environment, config_defaults=None):
+def instantiate_class_from_config(
+    config: dict,
+    runtime_environment: Optional[dict],
+    config_defaults: Optional[dict] = None,
+) -> Any:
     """Build a GX class from configuration dictionaries."""
 
     if config_defaults is None:
         config_defaults = {}
 
-    config = copy.deepcopy(config)
+    print(f"# BEGIN config {type(config)} ->\n{pf(config)}\n# END CONFIG")
+    config = copy.deepcopy(config)  # TypeError trying to pickle `module`
```

Comment:
This is where things break.

https://stackoverflow.com/questions/22388854/relationship-between-pickle-and-deepcopy